From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Tue, 17 Dec 2019 19:10:30 +0100
Subject: ANDROID: SoC: core: export snd_soc_info_multi_ext as gpl-only

The commit with change-id I26343af2e4a35952d93dc2d5fa0caad2a3e50ffe only
exported a new symbol, it needs to be marked as gpl-only until it lands
upstream to be properly handled.

Cc: Banajit Goswami <bgoswami@codeaurora.org>
Cc: Sudheer Papothi <spapothi@codeaurora.org>
Cc: Meng Wang <mwang@codeaurora.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I07bbe701dfcd3fe84818bde45ccac1cf699fe769
---
 sound/soc/soc-core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/soc-core.c b/sound/soc/soc-core.c
--- a/sound/soc/soc-core.c
+++ b/sound/soc/soc-core.c
@@ -3099,7 +3099,7 @@ int snd_soc_info_multi_ext(struct snd_kcontrol *kcontrol,
 	uinfo->value.integer.max = platform_max;
 	return 0;
 }
-EXPORT_SYMBOL(snd_soc_info_multi_ext);
+EXPORT_SYMBOL_GPL(snd_soc_info_multi_ext);
 
 int snd_soc_get_dai_name(struct of_phandle_args *args,
 				const char **dai_name)
